src/api/middleware/files.js

```diff
@@ -0,0 +1,110 @@
+const HTTPStatus = require("http-status-codes");
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const { MulterError } = require("multer");
+const multerConfig = require("../../config/multer");
+const { ErrorTypes } = require("./errorHandler");
+const cloudinary = require("cloudinary").v2;
+const env = require("../../config/env");
+const ValidationReasons = require("./validators/validationReasons");
+
+
+const save_folder = path.join(__dirname, "../../../public/");
+if (!fs.existsSync(save_folder)) fs.mkdirSync(save_folder);
```

Comment:
I think this should be on the loader (perhaps the ./static one) instead of being here dangling and possibly be called multiple times (even though it shouldn't be a problem, it make more sense to be in a loader)

Reply:
In order to access the save_folder constant in both places, you can declare it in the config (env stuff) and require it from both places